src/db/seeders/20190828065829-feedback-seeder.js

```diff
@@ -14,5 +14,5 @@ module.exports = {
     updated_at: new Date()
   }], {}),
 
-  down: (queryInterface) => queryInterface.bulkDelete('ratings', null, {})
+  down: (queryInterface) => queryInterface.bulkDelete('feedbacks', null, {})
```

Comment:
You combined the ratings and feedbacks table?

Reply:
There was a `typo` when the seeder was created; I accidentally named it `ratings` instead of `feedbacks`. Working on the feedback now, brought out the error, which has been corrected.